### DISCUSSION TITLE
Request for autosel timeout in Auto Installation plugin

### DISCUSSION THREAD

**Discussion by devdevadev** [ventoy/Ventoy#1161] [Ideas]
Hi LongPanda...

Please allow to use timeout of x seconds in Auto Install plugin if autosel option is used.

This way we have 5 sec to select different XML. If user does not select any XML within 5 sec then use given autosel number to auto select XML file.

"auto_install": [
        {
            "image": "/abc/centos.iso",
            "template": "/ventoy/script/centos_kickstart.cfg"
        },
        {
            "image": "/ISO/cn_windows_server_2012_r2_vl_x64_dvd_2979220.iso",
            "template": [
                "/ventoy/script/windows_unattended1.xml",
                "/ventoy/script/windows_unattended2.xml",
                "/ventoy/script/windows_unattended3.xml",
                "/ventoy/script/windows_unattended4.xml",
                "/ventoy/script/windows_unattended5.xml",
                "/ventoy/script/windows_unattended6.xml"
            ],
            "autosel": 2
            "timeout": 5
        },

Is it possible ?

**Comment by ventoy** [comment]
Please try this CI release:
https://github.com/ventoy/Ventoy/actions/runs/1362792594

**Comment by ventoy** [comment]
@devdevadev

**Reply by devdevadev** [reply]
Thanks a lot.....
